### PR TITLE
CreativeTechnologyLtd.SoundBlasterCommand version 3.4.92.00

### PR DESCRIPTION
**Pull request by mddtest**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17549)

### CODE REVIEW

**Comment by mddtest** [comment]
image

File: manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+PackageVersion: 3.4.92.00
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://files.creative.com/manualdn/Applications/100330/dccAqktyLO/AECMDMasterInstaller_3.4.92.00.exe
+  InstallerSha256: DE40104936219BD1AE32304405E1144F07CD6B81CD1D219CF2BACC7991B70BE3
+ManifestType: installer
+InstallerSwitches:
+    Silent: /silent
+    SilentWithProgress: /silent
```
Comment (OfficialEsco) [inline manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.installer.yaml:13]:
```suggestion
InstallerSwitches:
    Custom: /NORESTART
```

Comment (OfficialEsco) [reply manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.installer.yaml:13]: You don't need to add Silent switches when the installer type is inno/nullsoft, also this switch shoould work if i typed it correctly for inno

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: CreativeTechnologyLtd.SoundBlasterCommand
```
Comment (OfficialEsco) [inline manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.installer.yaml:3]:
```suggestion
PackageIdentifier: CreativeTechnology.SoundBlasterCommand
```

File: manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: CreativeTechnologyLtd.SoundBlasterCommand
```
Comment (OfficialEsco) [inline manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: CreativeTechnology.SoundBlasterCommand
```

File: manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: CreativeTechnologyLtd.SoundBlasterCommand
```
Comment (OfficialEsco) [inline manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.yaml:3]:
```suggestion
PackageIdentifier: CreativeTechnology.SoundBlasterCommand
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Didn't check the PackageID when i suggested the change, please remove Ltd from the PackageID
Remember to change the path/file name

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**